sycl/cmake/modules/FetchUnifiedRuntime.cmake

```diff
@@ -25,10 +25,12 @@ set(SYCL_UR_SOURCE_DIR
 
 option(SYCL_UMF_DISABLE_HWLOC
   "Disable hwloc support in UMF" ON)
+option(SYCL_UR_BUILD_TESTS "Build tests for UR" OFF)
```

Comment:
This is default off. Do we want to build UR tests as a default part of building LLVM & SYCL?

Reply:
We have a dedicated job for running unified-runtime tests which will probably stay for the time being, so I'm not sure if there's any point in enabling it as part of intel/llvm.

However, on the other hand, there's probably no reason to NOT disable it on CI either, so I'll just have it be included as part of ci-defaults.

Reply:
Some testing (before the nodes exploded) seems to bring up some problem with building UR tests as part of an intel/llvm job. I'll leave it disabled for now, and enable it in a separate PR (as well as fix the issue).